src/sentry/db/models/base.py

```diff
@@ -189,6 +257,13 @@ def __model_class_prepared(sender: Any, **kwargs: Any) -> None:
             f"like Group."
         )
 
+    from .outboxes import ProcessUpdatesWithControlOutboxes, ProcessUpdatesWithRegionOutboxes
+
+    if issubclass(sender, ProcessUpdatesWithControlOutboxes):
+        sender.__category__.connect_control_model_updates(sender)
```

Comment:
This is the other crucial bit -- while I can't control the metaclass to do class level initialization, django orm DOES provider this post model loading signal that lets me one time attach the model to a signal receiver on bootup.

Effectively, every class specifies which outbox is associated with the "default" model update category.  This does not preclude the model from creating outboxes that are not associated with updates.  Those outboxes must still be manually wired up.